app/assets/stylesheets/widgets/home_page/hero_panel.scss

```diff
@@ -21,7 +21,7 @@
   clear: both;
   position: relative;
   overflow: hidden;
-  min-height: 300px;
+  height: 300px;
```

Comment:
These min-height settings could explain it, no? They set a lower bound, but not an upper. So for images, or inner html, that exceeded that height the slide element might expand vertically.

Not sure why it started happening all the sudden. But perhaps something got tweaked in the way the flexslider manhandles the DOM.